### ISSUE TITLE
Implementation of login part from server side as well as client side of any person (parent/user/orphanage) 

### ISSUE THREAD

**Issue by Harshita248** [sanscript-tech/orphan_support-php#39]
After one gets registered he/she should be notified as 'already registered' so that one can login easily. Implementation of forgot password needs to be there as well.

**Comment by SanjeebNath382** [comment]
Hey, so the person who is making the login or sign-up page, can he add the relevant scripts and database?

**Comment by Harshita248** [comment]
Yes you can @SanjeebNath382 but first you create the signup and login pages for the website and work according to that pr. This can be assigned later . Focus on one issue which is assigned to you. After completion I may assign this to you .

**Comment by punitsakre23** [comment]
I like to work on this.

**Comment by Harshita248** [comment]
Okay go ahead @punitsakre23  . In case of any doubts ask any of the mentors.

**Comment by ujwal111** [comment]
Hi, I wanted to bring this to your notice, that some part of the issue is already implemented by me on 29th October, please refer the orphansupport.sql file, the entire database part is done, but not the login part.

**Comment by Harshita248** [comment]
Yes @ujwal111  that's why I added login too beside it. Rest it's okay I have updated it so you can go ahead if you want

**Comment by Harshita248** [comment]
@punitsakre23 when will this be done ?

**Comment by Harshita248** [comment]
@punitsakre23 need some response from you . So that I can assign it to someone else. Please reply asap
